core/create_spoken_forms.py

```diff
@@ -39,7 +39,7 @@
 )
 
 REVERSE_PRONUNCIATION_MAP = {
-    **{value: key for key, value in abbreviations.items()},
+    **{value: key for key, value in get_abbreviations_list().items()},
```

Comment:
won't this cause us to parse the abbreviations csv twice, both on initial load and when it's updated by the user? any reason not just to return ctx.lists["user.abbreviation"] for use externally?

I'm not entirely sure it matters, but figured I'd bring it up.